frontends/web/src/locales/en/app.json

```diff
@@ -833,13 +833,33 @@
       }
     },
     "settings-electrum": {
+      "connection": {
+        "text": "If you  intend to only connect to your node when you are on the same network (e.g. your home wifi), then using regular network communication is sufficient.\nBe aware that your Electrum server needs to provide a TLS certificate to encrypt the communication in that case.\nIf you intend to connect to your node from anywhere, using Tor is the better option. No TLS certificate is necessary in that case.",
+        "title": "Should I use clearnet or Tor?"
+      },
+      "instructions": {
+        "link": {
+          "text": "Guide to connect your node",
+          "url": "https://guides.shiftcrypto.ch/bitboxapp/full-node/"
+        },
+        "text": "For a full tutorial, please visit our guide:",
+        "title": "How do I connect my BitBoxApp to my own fullnode?"
+      },
+      "options": {
+        "text": "There are multiple options to run your own node such as buying a finished device, building your own or running Bitcoin Core.\nIf you want to connect your BitBoxApp to your node, make sure that it runs an Electrum server. This is a dedicated program that allows a wallet apps to communicate with your full node.\nSupported options include Electrs, Electrum Personal Server (EPS) or Bitcoin Wallet Tracker (BTW).",
```

Comment:
> or running Bitcoin Core

`or running an Electrum server`, then you can delete the next two sentences. In general maybe try to make the texts more concise.

Reply:
This text was already reviewed by Stadicus before I made the PR. I think it's good ;)